guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -4081,13 +4093,15 @@ WEBVTT
 [Stephen Frost]: And then Aldous Huxley comes along and writes
 
 02:25:09.278 --> 02:25:11.599
-[Stephen Frost]: Brave New World, which is his brother, right?
+[Stephen Frost]: Brave New World, 
+
```

Comment:
We might have to fix this later. (e.g. merge the changes first and then make this fix so there's less changes to deal with/review)

The line above shows up in the player, but not the line below and I don't know if it's because in a VTT file it expects 1 "timestamp" and then 1 "text" line followed by a blank line as the format.

It's possible if we get rid of this space, that the lines will show up together, but if that doesn't work, then we should actually add a timestamp to the line below.

The cheat might be to use the exact same timestamp as I've seen VTT/SRT files do that and both subtitle lines will show up (e.g. overlapping speech where one comes in a little earlier and leaves a little earlier, but they overlap for a period of time).

Reply:
I wondered about that. I wasn't sure how to add a timestamp.